crates/rb-sys/Cargo.toml

```diff
@@ -25,3 +26,5 @@ doctest = false
 [build-dependencies]
 bindgen = "0.59"
 pkg-config = "0.3.24"
+cc = "1.0"
+shell-words = "1.1.0"
```

Comment:
Can these be optional and only activated when the `ruby-macros` feature is enabled?

Off the top of my head I think the syntax would be

```
[features]
ruby-macros = ["cc", "shell-words"]

[build-dependencies]
cc = { version = "1.0", optional = true }
shell-words = { version = "1.1.0", optional = true }
```